[enterprise-4.19] OSDOCS-15477: add gwapi aws known issue (#96578)
openshift/openshift-docs: release_notes/ocp-4-19-release-notes.adoc

+
(link:https://issues.redhat.com/browse/OCPBUGS-52367[OCPBUGS-52367])

> * There is a known issue with Gateway API that prevents the provisioning of internal load balancers on {aws-first}, {gcp-first}, and {azure-first} private clusters. Load balancers configured for internal use are incorrectly determined to be external:

Reviewer: It is confusing to say, "Load balancers configured for internal use are incorrectly determined to be external"; the issue is that we don't provide an option to configure the load balancer to be internal.  
```suggestion
* There is a known issue with Gateway API and {aws-first}, {gcp-first}, and {azure-first} private clusters.  The load balancer that is provisioned for a gateway is always configured to be external, which can cause errors or unexpected behavior:
```

Author: thanks Miciah, i've updated the text

+
--
** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.

> ** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned, but the existence of an external IP address is unexpected.

Reviewer 2: ```suggestion
** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned, but an external IP address is  provisioned, which is unexpected for an internal load balancer.
```

Author: this is a great suggestion that adds clarity, thank you, i'll update the text

> ** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned with an external IP address. This is unexpected for an internal load balancer.

Reviewer 2: 🤷🏼‍♀️

+
(link:https://issues.redhat.com/browse/OCPBUGS-52367[OCPBUGS-52367])

> * There is a known issue with Gateway API that prevents the provisioning of an internal Gateway Load Balancer service on {aws-first} private clusters. (link:https://issues.redhat.com/browse/OCPBUGS-57440[OCPBUGS-57440])

Author: thank you, i've updated the note

> * There is a known issue with Gateway API that prevents the provisioning of an internal Gateway Load Balancer service on {aws-first} private clusters. (link:https://issues.redhat.com/browse/OCPBUGS-57440[OCPBUGS-57440])

Reviewer 3: Looks great! Thanks
